clients/vscode/src/lsp/WorkspaceFeature.ts

```diff
@@ -39,10 +39,6 @@ export class WorkSpaceFeature extends EventEmitter implements StaticFeature {
     this.disposables = [];
   }
 
-  get isAvailable(): boolean {
-    return !!this.registration;
-  }
-
```

Comment:
I removed some unused code. They seem to be related to dynamic registration but are no longer in use.